daemon/daemon.go

```diff
@@ -25,6 +26,7 @@ type Daemon struct {
 	systemMgr      mgr.SystemMgr
 	imageMgr       mgr.ImageMgr
 	volumeMgr      mgr.VolumeMgr
+	criManager     *cri.CRIManager
```

Comment:
To keep consistent, criMgr and CRIMgr?

Reply:
Actually, in other cases, the type of *Manager is struct and *Mgr is interface.

From this perspective, "CRIManager" is more consistent 😄 , but "CRIManager" seems can't pass go lint which said it is too stutter.

I'll change it to criMgr.